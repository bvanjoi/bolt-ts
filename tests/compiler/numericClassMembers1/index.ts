// From `github.com/microsoft/TypeScript/blob/v5.9.2/tests/cases/compiler/numericClassMembers1.ts`, Apache-2.0 License

class C234 {
  0 = 1; 
  0.0 = 2;
  //~^ ERROR: Duplicate identifier '0'.
}
 
class C235 { 
  0.0 = 1;
 '0' = 2;
  //~^ ERROR: Duplicate identifier '0'.
}

class C236 {
    '0.0' = 1;
    '0' = 2;
}
