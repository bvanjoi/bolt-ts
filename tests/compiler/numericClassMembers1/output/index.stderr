  × Duplicate identifier '0'.
   ╭─[../../tests/compiler/numericClassMembers1/index.ts:5:3]
 5 │ 0.0
   · ───
   ╰────

  × Duplicate identifier '0'.
    ╭─[../../tests/compiler/numericClassMembers1/index.ts:11:2]
 11 │ '0'
    · ───
    ╰────
